instrumentation/logback/logback-mdc-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/logback/mdc/v1_0/LoggingEventInstrumentation.java

```diff
@@ -69,11 +79,16 @@ public static void onExit(
         return;
       }
 
-      SpanContext spanContext = Java8BytecodeBridge.spanFromContext(context).getSpanContext();
+      Span span = Java8BytecodeBridge.spanFromContext(context);
+      SpanContext spanContext = span.getSpanContext();
       if (!spanContext.isValid()) {
         return;
       }
 
+      if (span.isRecording()) {
```

Comment:
why we are setting the attribute both on method enter and on method exist?

Reply:
My bad, removing the one on the onEnter